packages/nodes-base/nodes/Hubspot/Hubspot.node.ts

```diff
@@ -923,6 +938,82 @@ export class Hubspot implements INodeType {
 				return returnData.sort((a, b) => a.name < b.name ? 0 : 1);
 			},
 
+			/* -------------------------------------------------------------------------- */
+			/*                               Custom objects                               */
+			/* -------------------------------------------------------------------------- */
+
+			// Get all the custom object types to display them to user so that he can select them
+			async getCustomObjectTypes(this: ILoadOptionsFunctions): Promise<INodePropertyOptions[]> {
+				const returnData: INodePropertyOptions[] = [{
+					name: 'Contact',
+					value: '0-1',
+				}];
```

Comment:
The schemas endpoint does only return custom objects, but the API documentation states that the objects API can also be used with contacts and other HubSpot objects. I did not find a Hubspot api endpoint, that gave me all objects, so I just hardcoded contact for now. You can generate the full list of possible objects with something like `seq 50 | xargs -n1 -i{} curl 'https://api.hubapi.com/crm/v3/schemas/0-{}?hapikey=INSERT_API_KEY_HERE' 2>/dev/null | jq '{name: .labels.singular, value: .objectTypeId }'`.

It would probably be best to add the full list here, as that would allow generic access to all HubSpot objects, but it would also clutter the list and make the custom objects harder to find.

Reply:
I added the complete list of all object types that are valid to use with the `crm/v3/objects` api. The `crm/v3/schemas` endpoint supports all object types, a different list should be used for that.